charts/gha-runner-scale-set/templates/autoscalingrunnerset.yaml

```
@@ -21,6 +21,18 @@ spec:
   runnerScaleSetName: {{ . }}
   {{- end }}
 
+  {{- if .Values.githubServerTLS }}
+  githubServerTLS:
+    {{- if .Values.githubServerTLS.certificateFrom }}
+    {{- with .Values.githubServerTLS.certificateFrom }}
+    certificateFrom:
+      configMapKeyRef:
+        name: {{ .configMapKeyRef.name }}
+        key: {{ .configMapKeyRef.key }}
+    {{ end }}
+    {{ end }}
```

```suggestion
    {{- with .Values.githubServerTLS.certificateFrom }}
    certificateFrom:
      configMapKeyRef:
        name: {{ .configMapKeyRef.name }}
        key: {{ .configMapKeyRef.key }}
    {{ end }}
```

`with` should check empty for you already.